xapian-applications/omega/docs/omegascript.rst

```diff
@@ -341,14 +341,14 @@ $match{REGEX,STRING[,OPTIONS]}
 	peform a regex match using Perl-compatible regular expressions. This
 	command will return true if a match is found
 
-		The optional OPTIONS argument is supported by Omega 1.3.4 and later.
-    	It can contain zero or more of the letters ``imsx``, which have the
-    	same meanings as the corresponding Perl regexp modifiers:
-
-    	 * ``i`` - make the pattern matching case-insensitive
-     	 * ``m`` - make ``^``/``$`` match after/before embedded newlines
-     	 * ``s`` - allows ``.`` in the pattern to match a linefeed
-     	 * ``x`` - allow whitespace and ``#``-comments in the pattern
+	The optional OPTIONS argument is supported by Omega 1.3.4 and later.
+    It can contain zero or more of the letters ``imsx``, which have the
+    same meanings as the corresponding Perl regexp modifiers:
+
+    * ``i`` - make the pattern matching case-insensitive
+    * ``m`` - make ``^``/``$`` match after/before embedded newlines
+    * ``s`` - allows ``.`` in the pattern to match a linefeed
+    * ``x`` - allow whitespace and ``#``-comments in the pattern
```

Comment:
This alignment is still wrong; I think all the lines should be indented by one tab. Can you run the `xapian-maintainer-tools/xapian-check-patch` script over your patch and correct anything that complains about, as well please?